examples/persistence_mongodb.ipynb

```diff
@@ -0,0 +1,490 @@
+       "  ToolMessage(content=\"It's always sunny in sf\", name='get_weather', id='21c3d817-21dc-4a46-994a-4377928bdba7', tool_call_id='call_7zZnmZp7FwpLrTgH0iIvQOfb'),\n",
+       "  AIMessage(content='The weather in San Francisco is always sunny!', response_metadata={'token_usage': {'completion_tokens': 10, 'prompt_tokens': 86, 'total_tokens': 96}, 'model_name': 'gpt-3.5-turbo-0125', 'system_fingerprint': None, 'finish_reason': 'stop', 'logprobs': None}, id='run-03359fb4-2f8f-452c-b390-bd0ebe739bb6-0', usage_metadata={'input_tokens': 86, 'output_tokens': 10, 'total_tokens': 96}),\n",
+       "  HumanMessage(content=\"what's the weather in sf\", id='3cdd99cd-d82a-44b4-8ae1-3babbfb971ba'),\n",
+       "  AIMessage(content='', additional_kwargs={'tool_calls': [{'id': 'call_eGxogdSsCQRBcOt0IEoHAj71', 'function': {'arguments': '{\"city\": \"sf\"}', 'name': 'get_weather'}, 'type': 'function'}, {'id': 'call_f94gZnV1qP4j4dky3TXYgSQD', 'function': {'arguments': '{\"city\": \"sf\"}', 'name': 'get_weather'}, 'type': 'function'}]}, response_metadata={'token_usage': {'completion_tokens': 43, 'prompt_tokens': 109, 'total_tokens': 152}, 'model_name': 'gpt-3.5-turbo-0125', 'system_fingerprint': None, 'finish_reason': 'tool_calls', 'logprobs': None}, id='run-6d29a1f5-c252-4a90-b30c-6332ac62516e-0', tool_calls=[{'name': 'get_weather', 'args': {'city': 'sf'}, 'id': 'call_eGxogdSsCQRBcOt0IEoHAj71'}, {'name': 'get_weather', 'args': {'city': 'sf'}, 'id': 'call_f94gZnV1qP4j4dky3TXYgSQD'}], usage_metadata={'input_tokens': 109, 'output_tokens': 43, 'total_tokens': 152}),\n",
+       "  ToolMessage(content=\"It's always sunny in sf\", name='get_weather', id='92afc46c-23d2-45f0-b7d7-99345c226bd1', tool_call_id='call_eGxogdSsCQRBcOt0IEoHAj71'),\n",
+       "  ToolMessage(content=\"It's always sunny in sf\", name='get_weather', id='e6400534-18cc-48e7-a9cc-b7339018f512', tool_call_id='call_f94gZnV1qP4j4dky3TXYgSQD'),\n",
+       "  AIMessage(content='The weather in San Francisco is always sunny!', response_metadata={'token_usage': {'completion_tokens': 10, 'prompt_tokens': 178, 'total_tokens': 188}, 'model_name': 'gpt-3.5-turbo-0125', 'system_fingerprint': None, 'finish_reason': 'stop', 'logprobs': None}, id='run-30339777-bbe8-48fa-8f15-b9be9c3d34ab-0', usage_metadata={'input_tokens': 178, 'output_tokens': 10, 'total_tokens': 188})]}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "res"
+   ]
+  },
+  {
+   "attachments": {
```

Comment:
is there a way to show this without a screenshot? just by showing these outputs programmatically?

Reply:
done